### PR TITLE
Add CMP param to Formstack wrappers

### PR DESCRIPTION
**Pull request by kenlim**
Turns out we also serve Formstack forms through a proxy.

### CODE REVIEW

File: common/app/views/support/CmpParamCleaner.scala [hunk 1]
```diff
@@ -4,12 +4,17 @@ import org.jsoup.nodes.{Document, Element}
...
   override def clean(document: Document): Document = {
     val formStackHost: String = "guardiannewsampampmedia.formstack.com"
+    val profileFormstackWrapper: String = "profile.theguardian.com/form/embed"
 
     document.getElementsByAttributeValueContaining("src", formStackHost).foreach { elem: Element =>
-      elem.addClass("element-formstack")
+      elem.addClass("element-pass-cmp")
+    }
+
+    document.getElementsByAttributeValueContaining("src", profileFormstackWrapper).foreach { elem: Element =>
```
Comment (phamann) [inline common/app/views/support/CmpParamCleaner.scala]: Could you not just pass a regex to this to be more DRY?

File: facia-tool/public/package.json [hunk 2]
```diff
@@ -8,17 +8,17 @@
     "configFile": "js/jspm-config.js",
     "dependencies": {
       "EventEmitter": "npm:wolfy87-eventemitter@4.2.11",
-      "css": "^0.1.0",
-      "font-awesome": "4.2.0",
-      "jquery": "2.1.3",
+      "css": "github:systemjs/plugin-css@^0.1.0",
+      "font-awesome": "npm:font-awesome@4.2.0",
+      "jquery": "github:components/jquery@2.1.3",
```
Comment (phamann) [inline facia-tool/public/package.json]: What is going on here?

Comment (phamann) [inline facia-tool/public/package.json]: /cc @piuccio

**Comment by phamann** [comment]
:+1: Nice.

File: facia-tool/public/package.json (hunk 2 above)
Comment (piuccio) [inline facia-tool/public/package.json]: I suspect that it's a change in JSPM. I've seen it in facia-tool as well after installing JSPM 0.15. They used to have their own registry but apparently now they simply alias to the either npm or github.

Comment (piuccio) [inline facia-tool/public/package.json]: lol just realised it is facia-tool. If github starts behaving, teamcity will build #8731, I'll merge that, you can rebase/merge and it'll go away.

**Pull request merged**